batchflow/models/tf/unet.py

```diff
@@ -59,14 +57,32 @@ class UNet(EncoderDecoder):
     def default_config(cls):
```

Comment:
We need default values for filters here, so one could use `body` of this class in custom classes.

Reply:
There are other models where filters are not set in default_config, for example ResNet. I see 2 options: 
1) if one decides to use some class's body, they have to provide all needed parameters explicitly
2) call `build_config` in `fill_params`